keyboards/handwired/nortontechpad/info.json

```diff
@@ -0,0 +1,52 @@
+{
+    "keyboard_name": "NortonTechPad",
+    "manufacturer": "NortonTech",
+    "url": "",
+    "maintainer": "NortonTech",
+    "usb": {
+        "vid": "0xFEED",
```

Comment:
VIA will reject a keyboard using FEED as its vendorID

Reply:
Do you have any suggestions how to get a better vendorID